src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,49 @@
+"use client"
+import React, { useState } from 'react';
+import styles from './styles.module.scss';
+import mockData from '../__mock__/data.json';//aqui importamos el data.json para hacer pruebas
+
+const Carousel = () => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  //funciones para funcionamiento del carrusel
+  const handlePrevClick = () => {
+    setCurrentIndex((prevIndex) => (prevIndex > 0 ? prevIndex - 1 : 0));
+  };
+  const handleNextClick = () => {
+    setCurrentIndex((prevIndex) => (prevIndex < mockData.length - 3 ? prevIndex + 1 : 0));
+  };
+
+  return (
+    <div className={styles.carousel}>
+      <div className={styles.subtitulo}><h2>VOLÚMENES</h2></div>
+      <div className={styles.slider}>
+        
+        <button className={`${styles.arrow} ${currentIndex === 0 ? styles.disabled : ''}`}onClick={handlePrevClick}disabled={currentIndex === 0} >
+          <svg width="39" height="63" viewBox="0 0 39 63" fill="none" xmlns="http://www.w3.org/2000/svg"><g clipPath="url(#clip0_36_446)"><path d="M37.6446 0C25.306 10.3008 12.9653 20.6016 0 31.4258C12.7881 42.1127 25.2883 52.5553 37.7885 63C38.1916 62.6641 38.5969 62.3283 39 61.9924C33.8728 53.3077 28.9294 44.5096 23.5254 35.9972C21.4568 32.7388 21.3881 30.3986 23.4633 27.1162C28.863 18.5667 33.8152 9.74465 38.9468 1.03161C38.5127 0.689192 38.0787 0.344596 37.6446 0Z" fill="#178435"/></g>
+          <defs><clipPath id="clip0_36_446"><rect width="39" height="63" fill="white" transform="matrix(-1 0 0 1 39 0)"/></clipPath></defs>
+          </svg>
+        </button>
+        {mockData.slice(currentIndex, currentIndex + 3).map((item) => (          
+          <div key={item.volume} className={styles.carouselItem}>
```

Comment:
this kind of code is considered a bad practice, you need to separate the code into smaller components.
Create a new component called ItemVolumen.tsx an put into the new component this code.